ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm

```diff
@@ -22,6 +22,8 @@ SAPS
   ^=+
   ^Number\s+of
   ^\s*$$
+  # added to avoid errors with trucated values alert message
```

Comment:
Is this referring to `* indicates that the corresponding row element may have been truncated.`?